test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -61,7 +61,7 @@ UcxLog::UcxLog(const char* prefix, bool enable)
     }
 
     size_t str_len = strlen(str);
-    snprintf(str + str_len, sizeof(str) - str_len, "%d] ", getpid());
+    snprintf(str + str_len, sizeof(str) - str_len, "%d] ", ucs_get_tid());
```

Comment:
it will still break analyzer. maybe just remove?

Reply:
well yes, it will break until Anatoly fixes it.
in order for the analyzer to work with it, it needs to be in int3..

anyway, you prefer to remove it for now?

Reply:
yes